third_party_manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lastUpdated": "Thu Oct 06 2022 11:05:55 GMT-0400 (Eastern Daylight Time)",
+  "lastUpdated": "Wed Oct 12 2022 15:16:03 GMT-0400 (Eastern Daylight Time)",
```

Comment:
Why did the timestamp change but nothing else did?

Reply:
so when package.json changes the hooks for the third party update. since no deps changed the timestamp is the only thing that gets updated